uber/models.py

```diff
@@ -744,6 +744,34 @@ def affiliates(self):
                 'total': max(0, amt)
             } for aff, amt in sorted(amounts.items(), key=lambda tup: -tup[1])]
 
+        def insert_test_admin_account(self):
+            """
+            insert a test admin into the database with username "magfest@example.com" password "magfest"
+            this is ONLY allowed if no other admins already exist in the database.
+
+            :param session: database session object
+            :return: True if success, False if failure
+            """
```

Comment:
This function looks good and I din't think it needs to be changed, but I would like to make a general point about returning booleans.  Anytime you find yourself writing a function which returns ``True`` or ``False`` it's worth asking yourself whether there's something more meaningful that you could return instead of ``True``.  For example, a function which checks for an error could return an error string (which would be truthy) if an error occurs or return False/None/empty-string/etc if no error is found.

In this case I'd recommend returning False/None on failure and returning the actual ``AdminAccount`` on success.  It doesn't matter to the function which currently consumes this, but doing it this way has a couple of points in its favor:

1) There's no reason NOT to do it, since you can still write the client code the same way: ``if session.insert_test_admin_account()``

2) It makes writing unit tests easier, since it'll be easier to check the return value and then make assertions about the object you're about to insert.

3) If we ever have a new function that wants to make some addition/modification to the new objects before they're inserted, returning the object would make that a lot easier.

Anyway, not a huge deal and probably not worth changing before we merge this, but I've found that returning booleans is something I do sparingly, since there's almost always something better I can return that will accomplish the same thing but make future improvements easier.